docs/src/main/asciidoc/developer-guide/inc/internal/_ui.adoc

```diff
@@ -26,16 +26,25 @@ There are two document models contained within the ConfigModel object, both of t
 
 A single MappingDefinition model in the ConfigModel object stores information about field mappings and related lookup tables. Individual mappings are represented in instances of MappingModel, and lookup tables are represented by the LookupTable model.
 
-=== SERVICE
+==== SERVICE
 
 When the Data Mapper UI Bootstraps, a series of service calls are made to the mapping service (https://github.com/atlasmap/atlasmap/blob/master/ui/src/app/lib/atlasmap-data-mapper/services/mapping-management.service.ts[MappingManagementService]) and document service (https://github.com/atlasmap/atlasmap/blob/master/ui/src/app/lib/atlasmap-data-mapper/services/document-management.service.ts[DocumentManagementService]). 
 
 The document service is used to fetch our source/target document information (name of doc, fields). After these are parsed from the service, they are stored in the ConfigModel's inputDoc and outputDoc DocumentDefinition models.
 
 The mapping service is used to fetch our mappings for the fields mapped from the source to the target document. These mappings (and related lookup tables) are parsed by the management service and stored in the ConfigModel's mappings MappingDefinition model. 
 
-=== Mapping State
+=== @atlasmap/atlasmap
+TODO
+
+
+=== @atlasmap/standalone
+TODO
+
+== Mapping State
 
 As you add fields into mapping, mapping transitions to a different state. Here is a state machine diagram for current mapping state transition.
 
 image:AtlasMap_Mapping_State.svg[]
+
```

Comment:
Just grammar again: 'get' should be 'gets' or maybe it would be better as 'exits'.

Reply:
Thanks, fixed